controllers/doguExportManager.go

```diff
@@ -0,0 +1,147 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudogu/k8s-dogu-operator/v3/api/ecoSystem"
+	"github.com/cloudogu/k8s-dogu-operator/v3/controllers/resource"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"time"
+
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	k8sv2 "github.com/cloudogu/k8s-dogu-operator/v3/api/v2"
```

Comment:
I actually don't mind because this is purely an aesthetics thing, but the import grouping is wild!

I wonder if it is possible to set an import format syntax per-project?